style.css

```diff
@@ -0,0 +1,300 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+  max-width: 960px; 
+  margin: auto;
```

Comment:
The website would look closer to the real one if you remove line 5-6 and added `width:100%;` in your `body{}`